_faq/bugs-play-downloaded-videos.html

```diff
@@ -0,0 +1,17 @@
+---
+categories:
+- player
+- bugs
+type: info
+title: "Playing downloaded videos"
+---
+<section>
+    <p>To play downloaded videos a external video player is required.</p>
```

Comment:
```suggestion
    <p>In order to play downloaded videos, you need to use an external video player.</p>
```

We can argue about not using you to write directly to the person who clicked the entry but I would still change the order of the words as I did